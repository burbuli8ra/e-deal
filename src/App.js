import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import { Header } from './components';

const App = () => {
  return (
    <>
      <Router>
        <Header />
      </Router>
      {/* @todo replace with actual component */}
      <div>Main App Content</div>
    </>
  );
}

export default App;
