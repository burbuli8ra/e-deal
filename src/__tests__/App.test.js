import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

const setup = props => <App {...props} />;

describe('App Component', () => {
  let wrapper;
  const props = {};

  beforeEach(() => {
    wrapper = render(setup(props));
  });

  test('should match snapshot', () => {
    const { asFragment } = wrapper;
    expect(asFragment()).toMatchSnapshot();
  });
});