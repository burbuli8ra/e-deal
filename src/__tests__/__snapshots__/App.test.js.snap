// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should match snapshot 1`] = `
<DocumentFragment>
  <header
    class="css-3vq7ia"
    data-testid="header"
  >
    <a
      class="css-1rzlx2v"
      data-testid="header-logo"
      href="/"
    >
      <svg
        class="css-xzn57i"
        data-testid="header-icon"
      >
        logo.svg
      </svg>
      eDeal
    </a>
  </header>
  <div>
    Main App Content
  </div>
</DocumentFragment>
`;
